style.css

```diff
@@ -1,250 +1,530 @@
+
+.other__button {
+  margin-left: 0;
+  margin-top: 0;
 }
 
+.center__v {
+  display: flex;
+  align-items: center;
+  padding-left: 695px;
+}
+
+.middle {
+  display: flex;
+  flex-direction: column;
+}
+
+.middle__title {
+  position: relative;
+  color: red;
+  font-weight: 600;
+  font-size: 70px;
+  line-height: 85px;
+  right: 600px;
+  top: 100px;
+  margin: 0 auto;
+}
+
+.middle__image {
+  display: flex;
+  align-items: center;
+  margin-right: 5px;
+   
+}
+.middle__salon {
+  max-width: 100%;
+}
+
+
 .content {
+  position: relative;
+  width: 438px;
+  height: 898px;
+  background: url("./images/aside1.png");
+  margin: 0 auto;
+  left: 380px;
+  bottom: 889px;
+  
+}
+
+.content__box {
   display: flex;
-align-items: center;
-width: 400px;
-height: 800px;
-background: url("./images/aside1.png");
-margin-left: 1000px;
-margin-top: -900px;
-position: relative;
+  flex-direction: column;
+  margin: 0 auto;
 }
 
-.content__right  {
-  color: #ffffff;
-  margin-top: -200px;
-  margin-left: -300px;
+.content__right {
+  color: #fff;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  width: 316px;
+  height: 206px;
+  margin-top: -9px;
+  margin-left: 51px;
 }
 
 .content__title {
-  color: #ffffff;
-  margin-top: -600px;
-  margin-left: 110px;
+  margin-top: 51px;
+  margin-left: -20px;
+  color: #fff;
+  font-size: 30px;
+  line-height: 37px;
+  white-space: nowrap;
+}
+
+.content__smart {
+  margin-left: 50px;
+  margin-top: 36px;
 }
 
-.other__vector3 {
-display: flex;
-align-items: center;
+.content__wifi {
+  margin-left: 49px;
+  margin-top: 31px;
 }
-.other__vector4 {
-display: flex;
-align-items: center;
-margin-top: -15px;
-margin-left: 3px;
+
+.content__battery {
+  margin-left: 50px;
+  margin-top: 18px;
 }
-.other__testdrive {
-display: flex;
-align-items: center;
-margin-top: -12px;
-margin-left: 30px;
+
+.content__disk {
+  position: absolute;
+  margin-left: 48px;
+  margin-top: 805px;
 }
 
-.other__link_vector3 {
-display: inline-block;
-margin-top: 5px;
+.content__i {
+  position: absolute;
+  margin-left: 56px;
+  margin-top: 806px;
 }
 
-.other__center, .other__left {
-font-weight: 600;
-font-size: 35px;
-line-height: 43px;
-color: #000000;
+.content__text {
+  position: absolute;
+  margin-left: 78px;
+  margin-top: 808px;
 }
- .other__km, .other__sec {
-  font-size: 20px;
- }
-
- .other__bottom-l {
-display: flex;
-align-items: center;
-margin-left: 15px;
-margin-top: -20px;
-font-size: 10px;
- }
- .other__bottom-с {
+
+.content__click {
+  width: 160px;
+  height: 40px;
+  border: 2px solid #fff;
+  box-sizing: border-box;
+  color: #fff;
+  background: #e51937;
+  margin-left: 235px;
+  margin-top: 170px;
+}
+
+.content__snap {
+  display: flex;
+  align-items: center;
+  margin-left: 33px;
+}
+
+.container {
+  position: relative;
+  max-width: 438px;
+  height: 1000px;
+  background: url("./images/aside1.png");
+  bottom: 900px;
+}
+
+.container__aside {
+  display: flex;
+ flex-direction: column;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)